<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f0f0f0;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="topicLabel">
       <property name="text">
        <string>Alarm Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="choose_topic">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333; color: #333; border-radius: 5px; padding-left: 10px;</string>
       </property>
       <property name="placeholderText">
        <string>Enter alarm name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="choose_text">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333; color: #333; border-radius: 5px; padding: 5px;</string>
       </property>
       <property name="plainText">
        <string>Description</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCalendarWidget" name="choose_date">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
            /* Основной стиль календаря */
            QCalendarWidget {
                border: 1px solid #333; 
                color: #333; 
                border-radius: 5px;
                selection-background-color: #0078d7; 
                selection-color: white;
            }
            
            /* Ячейки с датами */
            QCalendarWidget QAbstractItemView {
                outline: 0;
                selection-background-color: #0078d7;
                selection-color: white;
            }

            /* Стиль для даты при наведении */
            QCalendarWidget QAbstractItemView::item:hover {
                background-color: #cfe2f3; /* Цвет при наведении */
                color: #333;
            }

            /* Стиль для выделенной даты */
            QCalendarWidget QAbstractItemView::item:selected {
                background-color: #0078d7;
                color: white;
            }

            /* Стиль заголовка */
            QCalendarWidget QHeaderView::section {
                background-color: #f0f0f0;
                color: #333;
                font-weight: bold;
            }
          </string>
         </property>
         <property name="gridVisible">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QCalendarWidget::SingleSelection</enum>
         </property>
         <property name="horizontalHeaderFormat">
          <enum>QCalendarWidget::LongDayNames</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="choose_time">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid #333; color: #333; border-radius: 5px; padding-left: 10px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="viewLabel">
       <property name="text">
        <string>View Friends:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="choose_unical">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333; color: #333; border-radius: 5px; padding-left: 10px;</string>
       </property>
       <item>
        <property name="text">
         <string>View friends</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No view friends</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="create_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
              border: 1px solid #333; 
              color: #333; 
              border-radius: 5px; 
              background-color: #e6e6e6;
          }
          QPushButton:hover {
              background-color: #d4d4d4;
          }
          QPushButton:pressed {
              background-color: #bcbcbc;
          }
        </string>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="home_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">
          QPushButton {
              border: 1px solid #333; 
              color: #333; 
              border-radius: 5px; 
              background-color: #e6e6e6;
          }
          QPushButton:hover {
              background-color: #d4d4d4;
          }
          QPushButton:pressed {
              background-color: #bcbcbc;
          }
        </string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
